etc/httpd/global.conf

```diff
@@ -951,3 +951,6 @@ RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/pocket(/?)$ /b/$1firefox/pocket$2 [P
 
 # bug 1136436
 RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/ios(/?)$ /b/$1firefox/ios$2 [PT]
+
+# bug 1205632
+RewriteRule ^/js/language/?$ /b/js/language/ [PT]
```

Comment:
That rule is a bit different, I know, but it's what I've been using when moving a redirect to bedrock.